src/Chapter1.hs

```diff
@@ -508,7 +521,8 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y =
+  let absn = min (abs x) (abs y) in if absn == abs x then x else y
```

Comment:
The implementation of this function is valid. But I see that it can be slightly improved. In your code, you use `abs` several time and introduce a helper variable, but in this task, it's enough to compare two absolute values only once to get the final result.

Reply:
oh yeah, sorry ! ```if (abs x) < (abs y) then x else y```  :facepalm: 

I must say, I wasn't expecting anyone to actually review the PR, thanks a lot !